docs/docsite/rst/user_guide/vault.rst

```diff
@@ -193,6 +193,19 @@ Result::
 
 See also :ref:`single_encrypted_variable`
 
+After you added the encrypted value to a var file, you can see the original value by using the debug module. Please note if your YAML file defines the `ansible_connection` variable (as we used in our example), it will take effect when you execute the command below. To prevent this, please make a copy of the file without the ansible_connection variable. 
+
+.. code-block:: console
+
+   cat vars.yml | grep -v ansible_connection >> vars_no_connection.yml
```

Comment:
useless use of cat

also .. connection does not affect debug .. so this step is really irrelevant

Reply:
The reason I included the step of getting rid of connection is that it actually didn't work when it was set to network_cli...Tested in 2.5. I was surprised too.

Reply:
addressed in 29410428b337e067e76c7df119b5036a0bc28343